mkdocs.yml

```diff
@@ -258,7 +258,7 @@ nav:
 #        - Categories: developer-guide/etendo-classic/concepts/Categories.md
 #        - Category:Concepts: developer-guide/etendo-classic/concepts/Category_Concepts.md
 #        - Category:Developers Guide: developer-guide/etendo-classic/concepts/Category_Developers_Guide.md
-#        - Client Side Development and API: developer-guide/etendo-classic/concepts/Client_Side_Development_and_API.md
+#        - Client Side Development and API: developer-guide/etendo-classic/concepts/client-side-development-and-api.md
```

Comment:
@Tomas-Vesco Rename the file developer-guide/etendo-classic/concepts/client-side-development-and-api.md as well.